notebooks/agent-breakpoints.ipynb

```diff
@@ -6,11 +6,9 @@
    "source": [
```

Comment:
<p>Please update the description according to the code:</p><ul><li>new method is <code>load_pipeline_snapshot</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>